.circleci/config.yml

```diff
@@ -737,6 +737,9 @@ jobs:
       - run:
           name: Install deps
           command: pip install -r docs/requirements.txt
+      - run:
+          name: Install package in editable mode
+          command: pip install -e .
```

Comment:
Why is this needed?

Reply:
It's fine, btw... I'm just skeptical of installing anything in editable mode in CI unless it's to test that editable mode works.

Reply:
It isn't - the docs/requirement.txt was a mess of niworkflows / documentation dependencies, so cleaned it up